android/src/app/organicmaps/MwmWork.java

```diff
@@ -0,0 +1,52 @@
+package app.organicmaps;
+
+import android.content.Context;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.work.Worker;
+import androidx.work.WorkerParameters;
+import app.organicmaps.util.CrashlyticsUtils;
+import app.organicmaps.util.log.Logger;
+
+public abstract class MwmWork extends Worker
```

Comment:
Is that base class really needed?

Reply:
currently it is not needed, in old code it was there, thats why I thought may be it is written to fullfill any longterm goal.

Reply:
should I remove it?